.github/workflows/pull-request.yaml

```diff
@@ -17,6 +17,13 @@ jobs:
         run: make ci-go-build-all-platforms
         timeout-minutes: 30
 
+      - name: Upload binaries
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: binaries
+          path: _release
+
```

Comment:
what is this doing?

Reply:
It'll upload the binaries build in the PR into Github's artifact store: for example [here](https://github.com/open-policy-agent/opa/actions/runs/792241966), at the very bottom:

![image](https://user-images.githubusercontent.com/870638/116397190-965f8c80-a826-11eb-8773-aaf1c186af37.png)